### ISSUE TITLE
Should I start completely over?

### ISSUE THREAD

**Issue by MindpowerLLC** [HashLips/hashlips_nft_minting_dapp#42]
Should I start from creating a completely new batch of NFTs from creation to mint?  Please take a moment to hear me out.

1) I used hashlips/scrawnyvikiny video and code in October.  made 10k/NFTs with 10k/jsons.  Done
2) Put them on IPFS, then CID to Pinata.  Paying $20 plan since my load was 9.4 gigs.
3) Made a Solidity contract using Remix.ethereum for polygon 137 - minted 100 to myself
4) I checked opensea to see my NFTs, only about 3/4s did NOT load images.  Just showing the collection avatar photo.
5) Checked back the next day - 4 new owners with 25 of my NFTs, I got nothing. - I don't know how.  I was robbed.
6) Freaked out
7) Went to IPFS and erased both JSON and IMAGE folders. - I figured Pinata has them and safe there, but it was public on IPFS.  **Am I wrong here?  Did I erase them from opensea?**
8) Started the process of creating a minting station on Github per SVikings guide. - never got it to work on github - worked on my local.  I didn't have enough matic  to mint one myself.  150 matic.
9) Tried using hashlips nft minting dapp and it all looks good from local, but have not tried on github because now I'm stuck at this new error - 

**Error: Invalid number of parameters for "mint". Got 1 expected 2!**

claimNFTs
http://localhost:3000/static/js/main.chunk.js:196:38

  193 | console.log("Gas limit: ", totalGasLimit);
  194 | setFeedback(`Minting your ${CONFIG.NFT_NAME}...`);
  195 | setClaimingNft(true);
> 196 | blockchain.smartContract.methods.mint(mintAmount).send({
      |                                  ^  197 |   gasLimit: String(totalGasLimit),
  198 |   to: CONFIG.CONTRACT_ADDRESS,
  199 |   from: blockchain.account,

onClick
src/App.js:359

  356 |   disabled={claimingNft ? 1 : 0}
  357 |   onClick={(e) => {
  358 |     e.preventDefault();
> 359 |     claimNFTs();
      | ^  360 |     getData();
  361 |   }}
  362 | >

I basically don't know WTH I'm doing.  I'm good with instructions, but some of the instructions did not work.  Please help, should I start all over?

**Comment by maxpar1** [comment]
Did you ever figure this out? I'm having the same bug and I'm not sure what to do.

**Comment by MindpowerLLC** [comment]
No, not yet, but working on it.  It has to do with using the wrong contract.  I am going to run the whole process all over again.  I may also see if I can get make website/app you can just use to add all you need and hit a button.  All these parameters can be added to a form that can be so much easier to use, but if it was that easy, your NFTs would be worthless.

**Comment by MindpowerLLC** [comment]
I did it all over again, works.  However, can't get the dapp to work on github.  I know I can read some of the info and go over the vids again, but I'm just getting burnt out from it all.  However, I know it will feel great when I figure it out.
